alchemy_news/README.md

```diff
@@ -0,0 +1,152 @@
+
+
+#### Configure Endpoints (WEX 11+)
+
+Two sample endpoints are provided, one for the text based list widget and another for the news sentiment trend widget.
+
+**News List Endpoint**
+
+This endpoint will search AlchemyData News and return summarized snippets of the top `N` news articles.
+
+1. From the Application Builder Admin console, navigate to the Endpoints section and create an Endpoint named `alchemy_news_list`.
+2. Add the following parameters with sample text to the endpoint.
+  - end
+      * now
+  - query
+      * |text=ibm|
+  - sentiment
+      * positive
+  - start
+      * now-7d
+  - max
+      * 5
+3. Copy the code from [news-list-endpoint.rb](appbuilder/alchemy-news-list-endpoint.rb)
+4. Update the API key in the endpoint to use your API key by replacing the text `YOUR_API_KEY_HERE`.
+5. Test the endpoint
+6. Save it.
+
+
+**News Sentiment Trend Endpoint**
+
+This endpoint will search AlchemyData News and return a computed histogram of news articles for a given time period so they can be graphed.
+
+1. From the Application Builder Admin console, navigate to the Endpoints section and create an Endpoint named `alchemy_news_trends`.
+2. Add the following parameters with sample text to the endpoint.
+  - end
+     * now
+  - query
+     * |text=ibm|
+  - sentiment
+     * positive
+  - start
+     * now-7d
+  - timeslice
+     * 1d
+3. Copy the code from [news-trend-endpoint.rb](appbuilder/alchemy-news-trend-endpoint.rb)
+4. Update the API key in the endpoint to use your API key by replacing the text `YOUR_API_KEY_HERE`.
+5. Test the endpoint
+6. Save it.
+
+
+
+#### Configuring the News Trend Widget
+
+The purpose of this example widget is to perform a AlchemyData News API search for the entity name and display a graphical trend of news sentiment.  The results of the AlchemyData News search are displayed in the Application Builder UI.  Controls are provided to modify the news search.
+
+Once you have logged into the Application Builder administrative interface, follow these steps to create the custom widget and add it to the Book Title entity.
+
+1. Navigate to the Pages & Widgets -> Book Title -> detail page.
+2. Create a new Custom widget.
+3. Set the ID of the widget to be `alchemy_news_api_list`
+4. Set the Display name of the widget to be `AlchemyData News Search`
+5. Copy and paste the [code for this widget](appbuilder/alchemy-news-list-widget.erb) into the Type-specific Configuration.
+6. Save the widget.
+7. Go back to the Book Title > detail page.
+8. Drag the `WDC_AlchemyAPI_News` widget to the top of the right column and save the page configuration.
+
+
+#### Configuring the News List Widget
```

Comment:
It looks like there is a bit of mix up between the "list" and "trend" widgets here in the descriptions, and in the specific steps below.